tgui/packages/tgui/interfaces/MessageMonitor.tsx

```diff
@@ -0,0 +1,338 @@
+              icon="home"
+              onClick={() => act('return_home')}
+            />
+          }>
+          <Table>
+            <Table.Row header>
+              <Table.Cell>Delete</Table.Cell>
+              <Table.Cell>Sender</Table.Cell>
+              <Table.Cell>Recipient</Table.Cell>
+              <Table.Cell>Message</Table.Cell>
+            </Table.Row>
+            {messages?.map((message) => (
+              <Table.Row key={message.ref}>
+                <Table.Cell>
+                  <Button
+                    icon="trash"
+                    color="red"
+                    onClick={() => act('delete_message', { ref: message.ref })}
+                  />
+                </Table.Cell>
+                <Table.Cell>{message.sender}</Table.Cell>
+                <Table.Cell>{message.recipient}</Table.Cell>
+                <Table.Cell>{message.message}</Table.Cell>
+              </Table.Row>
+            ))}
+          </Table>
+        </Section>
+      </Stack.Item>
+    </Stack>
+  );
+};
+
+const HackedScreen = (props, context) => {
+  return (
+    <Stack.Item grow>
+      <Stack fill vertical>
+        <Stack.Item grow />
+        <Stack.Item align="center" grow>
+          <Box color="red" fontSize="18px" bold mt={5}>
+            E%*#OR: CRIT#%^CAL Ss&@STEM Fai++~|URE...
+          </Box>
+          <Box color="red" fontSize="18px" bold mt={5}>
+            EN*bLING u&*E REB-00T ProT/.\;L...
+          </Box>
+        </Stack.Item>
+      </Stack>
+    </Stack.Item>
+  );
+};
+
+const MainScreen = (props, context) => {
+  const { act, data } = useBackend<Data>(context);
+  const { status, auth, server_status, is_malf, password } = data;
+  const [input_password, setPassword] = useLocalState(
+    context,
+    'input_password',
+    password
+  );
+  return (
+    <Stack fill vertical>
+      {server_status === 1 ? (
+        <>
+          <Stack.Item content="Welcome, please select an option" />
+          <Stack.Item>
+            <Section>
+              <Input
+                value={input_password}
+                onInput={(e, value) => setPassword(value)}
+                placeholder="Password"
+              />
+              <Button
+                content={auth ? 'Logout' : 'Auth'}
+                onClick={() => act('auth', { password: input_password })}
+              />
+              <Button
+                content={status ? 'ON' : 'OFF'}
+                color={status ? 'green' : 'red'}
+                disabled={!auth ? true : false}
+                onClick={() => act('turn_server')}
+              />
+              {is_malf ? (
+                <Button
+                  content="Hack"
+                  color="red"
+                  disabled={auth ? true : false}
+                  onClick={() => act('hack')}
+                />
+              ) : (
+                ''
+              )}
+            </Section>
+          </Stack.Item>
+        </>
+      ) : (
+        ''
```

Comment:
You can `{server_status == 1 && (` instead of using a ternary if the other side is just blank. Do this for everything

Reply:
Wow, thank you